bokeh/models/annotations.py

```diff
@@ -439,9 +439,10 @@ class Tooltip(Annotation):
     """
     level = Override(default="overlay")
 
-    side = Either(Auto, Enum(Side), default="auto", help="""
+    attachment = Enum("horizontal", "vertical", "left", "right", "above", "below", help="""
```

Comment:
> that we might want to automatically dodge multiple tooltips

This functionality doesn't belong to an annotation, rather to a tool or perhaps a manager on top of that. Still `auto` in this case may make sense, but to automatically choose between horizontal and vertical dimensions.